Makefile

```diff
@@ -41,7 +41,7 @@ psalm:
 	$(PSALM_BIN) analyse
 
 infection: tools-install composer-install
-	$(INFECTION_BIN) --threads=$(shell nproc || sysctl -n hw.ncpu || 1) --test-framework-options='--testsuite=Tests' --only-covered --min-msi=84
+	$(INFECTION_BIN) --threads=$(shell nproc || sysctl -n hw.ncpu || 1) --test-framework-options='--testsuite=Tests' --only-covered --min-msi=83
```

Comment:
I would like to avoid lowering the mutation score. Can you check if some of the introduced mutations can be covered by tests instead?

Reply:
The introduced mutation is because I am ignoring part of the output, therefore whatever the config for it is, it can be mutated freely. This is by the design of this change. Therefore the mutation score has to be increased on some other tests, not this one. And that change should not be part of this PR.